website/docs/d/devices.html.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
don't forget to add this file to https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/website/cloudflare.erb so that it is visible in the left hand navigation.